src/main.js

```diff
@@ -10,10 +10,10 @@ var saveCoverButton = document.querySelector('.save-cover-button');
 var homeButton = document.querySelector('.home-button');
 var viewSavedButton = document.querySelector('.view-saved-button');
 var savedViewPage = document.querySelector('.saved-view');
-var iputCoverImage = document.querySelector('#user-cover');
-var inputTitle = document.querySelector('#user-title');
-var inputDesc1 = document.querySelector('#user-desc1');
-var inputDesc2 = document.querySelector('#user-desc2');
+var inputCoverImage = document.querySelector('#cover');
+var inputTitle = document.querySelector('#title');
+var inputDesc1 = document.querySelector('#descriptor1');
```

Comment:
Good thing we opened the console or we would have never changed the id names to what the id's actually said.